docs/0001-negative-unl/README.md

```diff
@@ -0,0 +1,268 @@
+# Negative UNL Engineering Spec
+
+## The Problem Statement
```

Comment:
This document will be read in code editors and the command line (as well as formatted online). It would be good to reformat this to a max 80 column line limit.

Reply:
Agree, this is a good idea. 
Done.

Reply:
Thanks so much! Replaced with the patch.